files-np@4.0.0.0: download via aria2 failed

❯ scoop install files-np
WARN  Purging previous failed installation of files-np.
ERROR 'files-np' isn't installed correctly.
Removing older version (4.0.0.0).
'files-np' was uninstalled.
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Installing 'files-np' (4.0.0.0) [64bit] from 'nonportable' bucket
Starting download with aria2 ...
Download: 11/19 09:43:15 [ERROR] CUID#8 - Download aborted. URI=https://cdn.files.community/files/stable/Files.Package_4Download: Exception: [AbstractCommand.cc:351] errorCode=22 URI=https://cdn.files.community/files/stable/Files.Package_4.Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 30389f|ERR |       0B/s|D:/Application/Scoop/cache/files-np#4.0.0.0#c75c332.msixbundle
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

WARN  Download failed! (Error 22) HTTP response header was bad or unexpected
WARN  https://cdn.files.community/files/stable/Files.Package_4.0.0.0_Test/Files.Package_4.0.0.0_x64_arm64.msixbundle
    referer=https://cdn.files.community/files/stable/Files.Package_4.0.0.0_Test/
    dir=D:\Application\Scoop\cache
    out=files-np#4.0.0.0#c75c332.msixbundle

WARN  & 'D:\Application\Scoop\apps\aria2\current\aria2c.exe' --input-file='D:\Application\Scoop\cache\files-np.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.6 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=6168 --continue --summary-interval=0 --auto-save-interval=1 --all-proxy='127.0.0.1:7897'
WARN
Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Nonportable/issues/new?title=files-np%404.0.0.0%3a+download+via+aria2+failed
Fallback to default downloader ...
The remote server returned an error: (403) Forbidden.
URL https://cdn.files.community/files/stable/Files.Package_4.0.0.0_Test/Files.Package_4.0.0.0_x64_arm64.msixbundle is not valid

--- a/bucket/files-np.json
+++ b/bucket/files-np.json
@@ -1,4 +1,5 @@
 {
+    "##": "Please avoid removing the checkver.url property until https://github.com/ScoopInstaller/Scoop/pull/6490 has been merged into master.",
     "version": "4.0.21.0",
     "homepage": "https://files.community/",
     "description": "A modern file manager that helps users organize their files and folders.",
@@ -17,6 +18,7 @@
         "script": "Get-AppxPackage 'Files' | Remove-AppxPackage"
     },
     "checkver": {
+        "url": "https://github.com/files-community/Files",
         "script": [
             "try {",
             "  $latest = Invoke-WebRequest -Uri 'https://api.github.com/repos/files-community/Files/releases/latest'",

